source/stylesheets/card-back.scss

```diff
@@ -0,0 +1,21 @@
+.card--back{
+  z-index: $base-z-index;
+  color: $white;
+
+  .card--back-content {
+    -moz-transform: scaleX(-1);
+    -o-transform: scaleX(-1);
+    -webkit-transform: scaleX(-1);
+    transform: scaleX(-1);
+    filter: FlipH;
+    -ms-filter: "FlipH";
```

Comment:
2D transforms are very well supported, so we probably don't need all of these vendor fallbacks: https://caniuse.com/#feat=transforms2d

There's a cool tool called Autoprefixer [https://github.com/postcss/autoprefixer][https://github.com/postcss)/autoprefixer) that you can set up to do automatically add vendor prefixes along with compilation, if you'd like. We use it on most of our projects